src/styles/form.less

```diff
@@ -0,0 +1,98 @@
+// General Form Styles
+@-webkit-keyframes autofill {
+  to {
+    color: #666;
+    background: @white;
+  }
+}
+
+input:-webkit-autofill {
+  -webkit-animation-name: autofill;
+  -webkit-animation-fill-mode: both;
+}
+
+form {
+  margin: 0;
+}
+
+label {
+  user-select: none;
+}
+
+.radio-component,
+.checkbox-component {
```

Comment:
Would these be better included in their own less files? If I find a CSS issue with a checkbox, I wouldn't want to look in two files.